crates/sui-framework/packages/sui-framework/sources/deny_list.move

```diff
@@ -13,10 +13,37 @@ module sui::deny_list {
     const ENotSystemAddress: u64 = 0;
     /// The specified address to be removed is not already in the deny list.
     const ENotDenied: u64 = 1;
+    /// The specified address is cannot be added to the deny list.
+    const EInvalidAddress: u64 = 1;
 
     /// The index into the deny list vector for the `sui::coin::Coin` type.
     const COIN_INDEX: u64 = 0;
 
+    /// These addresses are reserved and cannot be added to the deny list.
+    /// The addresses listed are well known package and object addresses. So it would be
+    /// meaningless to add them to the deny list. As such, they are reserved for internal use.
+    const RESERVED: vector<address> = vector[
+        @0x0,
+        @0x1,
+        @0x2,
+        @0x3,
+        @0x4,
+        @0x5,
+        @0x6,
+        @0x7,
+        @0x8,
+        @0x9,
+        @0xA,
+        @0xB,
+        @0xC,
+        @0xD,
+        @0xE,
+        @0xF,
+        @0x403,
```

Comment:
bridge?

Reply:
Bridge package (@patrickkuo)
`const SUI_BRIDGE_ID: address = @0x9;`

Reply:
bridge object is 0x9 and bridge package is 0xb